source/core/slang-secure-crt.h

```diff
@@ -47,6 +47,7 @@ inline size_t strnlen_s(const char * str, size_t numberOfElements)
 #endif
 }
 
+__attribute__((format(printf, 3, 4)))
```

Comment:
Does msvc support this?

Reply:
haha, of course not. I'll hide it behind a macro...